client/src/i18n/en/report.json

```diff
@@ -141,6 +141,10 @@
       "DESCRIPTION": "This report shows the current stock value for each inventory in a depot",
       "EXCLUDE_INVENTORIES_ZERO_VALUE": "Exclude inventories with zero stock value"
     },
+    "STOCK_CONSUMPTION_GRAPH_REPORT" : {
+      "TITLE" : "Stock Consumption's graph report",
```

Comment:
This could be something like "Stock Consumption from {{depotName}} during {{periodLabel}}".

Reply:
Could you make this modification?  Or did you disagree with it?